DOCS/man/mpv.rst

```diff
@@ -1194,6 +1194,27 @@ PROTOCOLS
 
     Stitch together parts of multiple files and play them.
 
+``part://URL@start[-end]``
```

Comment:
Maybe it would be better to have start/end before the URL, to avoid escaping problems and such? (If an URL happens to contain a `@` character.)
Also I wonder if "slice" would be a better name than "part", but up to you.

Reply:
Moved start/end to before URL.

I agree that slice is probably a better name. Will change it after everything else is addressed.